src/Compilers/CSharp/Test/Syntax/Parsing/ValueTupleTests.cs

```diff
@@ -0,0 +1,494 @@
+using Xunit;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Parsing
+{
+    class ValueTupleTests : ParsingTests
```

Comment:
Good question. It didn't have trouble running from TestDriven.Net, but I didn't verify they were run from xUnit.